modules/virt-editing-vmis-web.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/cnv-manage-vmis.adoc
+
+[id="virt-editing-vmis-web_{context}"]
+= Editing a standalone virtual machine instance using the web console
+
+You can edit labels and  annotations for a standalone virtual machine instance (VMI) using the web console. Other items displayed in the *Overview* page for a standalone VMI are not editable.
```

Comment:
` Other items displayed in the *Overview* page for a standalone VMI are not editable.` It should be `Details`.

Reply:
Thanks for the correction. I verified this while reviewing the UI with @matthewcarleton. Made the correction to show the accurate sentence referring to "Details" page.

Reply:
Please review.

Reply:
Looks good.